go-controller/pkg/node/node.go

```diff
@@ -99,6 +99,68 @@ func setupOVNNode(node *kapi.Node) error {
 			return fmt.Errorf("error setting OVS encap-port: %v\n  %q", errSet, stderr)
 		}
 	}
+	if config.Monitoring.NetFlowTargets != nil {
+		collectors := ""
+		for _, v := range config.Monitoring.NetFlowTargets {
+			collectors += "\"" + v.Host.String() + ":" + strconv.Itoa(v.Port) + "\"" + ","
```

Comment:
you can change the type to hold int32, then you can directly use our util/net.go functions:

```
// JoinHostPortInt32 is like net.JoinHostPort(), but with an int32 for the port
func JoinHostPortInt32(host string, port int32) string {
	return net.JoinHostPort(host, strconv.Itoa(int(port)))
}

// SplitHostPortInt32 splits a vip into its host and port counterparts
func SplitHostPortInt32(vip string) (string, int32, error) {
	ip, portRaw, err := net.SplitHostPort(vip)
	if err != nil {
		return "", 0, err
	}
	port, err := strconv.ParseInt(portRaw, 10, 32)
	if err != nil {
		return "", 0, err
	}
	return ip, int32(port), nil
}
```

This will be more consistent with the rest of the ovn-k8s code.